.gitignore

```diff
@@ -1,7 +1,8 @@
-.gradle
-bin/
-build/
-dist/
-doc/
-README.html
-checkstyle/*.jar
+.gradle
+bin/
+build/
+dist/
+doc/
+README.html
+checkstyle/*.jar
+.classpath
```

Comment:
What was changed with all the other lines here? Line endings?

Reply:
Looks like it happened again